roles/datastore/datastore_setup/vars/main.yml

```diff
@@ -14,3 +14,8 @@ user_home_dir: /home/{{ custom_user }}/
 repo_dir: IATI-Datastore/
 
 wsgi_file_name: datastore.wsgi
+
+apache2_conf_file: apache2.conf
+
+# This variable may be refactored away later if it becomes cumbersome as the number of playbooks increases and we no longer wish to specify the version of mod-wsgi to use for each project
```

Comment:
Can the "become cumbersome" part be avoided by having a default value in the `apache2` role, and then overriding it if required?

Reply:
but what should the default be?